app/assets/javascripts/controllers/service/service_form_controller.js

```diff
@@ -0,0 +1,66 @@
+ManageIQ.angular.app.controller('serviceFormController', ['$http', '$scope', 'serviceFormId', 'miqService', function($http, $scope, serviceFormId, miqService) {
+    var init = function() {
+
+      $scope.serviceModel = {
+        name: '',
+        description: ''
+      };
+      $scope.formId = serviceFormId;
+      $scope.afterGet = false;
+      $scope.newRecord = false;
+      $scope.modelCopy = angular.copy( $scope.serviceModel );
+      ManageIQ.angular.scope = $scope;
+
+      miqService.sparkleOn();
+      $http.get('/service/service_form_fields/' + serviceFormId).success(function(data) {
+        $scope.serviceModel.name        = data.name;
+        $scope.serviceModel.description = data.description;
+
+        $scope.afterGet = true;
+        $scope.modelCopy = angular.copy( $scope.serviceModel );
+        miqService.sparkleOff();
+      });
+
+      $scope.$watch("serviceModel.name", function() {
+        $scope.form = $scope.angularForm;
```

Comment:
Why is this watch here? Seems like `$scope.form` is never used anywhere else, and `$scope.model` could just as well be set in `init`?